stl/inc/yvals_core.h

```diff
@@ -1207,7 +1207,9 @@
 #define __cpp_lib_integer_comparison_functions 202002L
 #define __cpp_lib_interpolate                  201902L
 #define __cpp_lib_is_constant_evaluated        201811L
+#define __cpp_lib_is_layout_compatible         202101L

```

Comment:
New feature-test macros need to be tested in `tests/std/tests/VSO_0157762_feature_test_macros`.

Reply:
Added new feature-test macros tests